track/mounts.py

```diff
@@ -619,6 +637,20 @@ def safe(self) -> bool:
         return True
 
 
+    def shutdown(self) -> bool:
+        """Bring mount into a safe state by stopping motion and shut down.
+
+        This method blocks until motion has ceased. Due to implementation details of the `Gemini2`
+        class it is important to call this method rather than (only) `safe()` before program end
+        to ensure that cleanup of running processes occurs.
+
+        Returns:
+            True when motion is stopped. Will not return otherwise.
+        """
+        self.mount.shutdown()
```

Comment:
- [ ] Should this change require updating the minimum version of the `point` package?